apps/api/src/domain/factories/__tests__/buildAggregateFactoryTestCases/common/buildNullAndUndefinedAggregateFactoryInvalidTestCases.ts

```diff
@@ -0,0 +1,17 @@
+import { AggregateFactoryInalidTestCase } from '..';
+import assertErrorAsExpected from '../../../../../lib/__tests__/assertErrorAsExpected';
+import NullOrUndefinedAggregateDTOError from '../../../../domainModelValidators/errors/NullOrUndefinedAggregateDTOError';
+import { AggregateType } from '../../../../types/AggregateType';
+
+export default (aggregateType: AggregateType): AggregateFactoryInalidTestCase[] =>
+    (
+        [
+            [null, 'null'],
+            [undefined, 'undefined'],
+        ] as const
+    ).map(([value, label]) => ({
+        description: `when the dto is ${label}`,
+        dto: value,
+        checkError: (result: unknown) =>
```

Comment:
too bad we can't make this callback pointfree. 
```
(result: unknown): =>
```
accounts for many, many lines in this PR!